client/src/app/metadata-schema-form/metadata-form/metadata-form.component.html

```diff
@@ -1,6 +1,6 @@
 <form *ngIf="done" class="vf-form" [formGroup]="metadataForm.formGroup" (submit)="onSubmit($event)">
   <ng-container *ngIf="visibleTabs.length > 1">
-    <app-vf-tabs *ngIf="useVf" (selectedIndexChange)="onSelectedIndexChange($event)">
+    <app-vf-tabs *ngIf="!!useVf" (selectedIndexChange)="onSelectedIndexChange($event)" [selectedIndex]="lookupTabIndex(selectedTabKey)">
```

Comment:
just curious, why do `!!`?

Reply:
Actually force of habit, not really necessary here. Ensures it's actually a boolean so e.g. `null` becomes `false`

Reply:
I see. Thanks!